.github/workflows/release-drafter.yml

```diff
@@ -13,8 +13,24 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Setup Yarn
+        run: |
+          npm uninstall -g yarn
+          npm i -g yarn@1.22.10
+          yarn install --frozen-lockfile
+      - name: Generate build
+        run: |
+          yarn build
+          yarn pack --filename oui.tar.gz
```

Comment:
Recently we release reporting0cli and they are creating the tarball using npm pack. See: https://github.com/opensearch-project/reporting-cli/blob/main/.github/workflows/release-drafter.yml#L17-L29
is that different from whet you are doing here?

Reply:
I don't think so. I think `yarn pack` passes through or emulates the functionality of `npm pack` while still being a yarn command. I think I'd prefer to stick with yarn commands as much as possible so we don't have any chance of worrying about conflicting `yarn.lock` and `package-lock.json` files